content/rancher-prime/aws/contents.lr

```diff
@@ -53,7 +53,7 @@ eksctl utils associate-iam-oidc-provider --cluster $CLUSTER_NAME --region $REGIO
 
 To provide the necessary permissions, an IAM role and an attached policy are required. The role name is passed as an argument during the *helm* deployment.
 
-Create the role with a *role name* of your choosing (for example, `rancher-csp-iam-role`), and the required policy attached to it:
+Create the role with a *role name* of your choosing (for example, `rancher-csp-iam-role`, export ROLE_NAME=rancher-csp-iam-role), and the required policy attached to it:
```

Comment:
I think I would format this line a little differently... its not entirely obvious to me that
`export ROLE_NAME=rancher-csp-iam-role` is a command meant for the user to be run. Maybe in its own code block or as a line in the block below just before the eksctl command... there's some options, but the current form isn't my favorite.